### ISSUE TITLE
Dev deve abrir ou não a câmera no trabalho remoto? 

### ISSUE THREAD

**Issue by alvarogfn** [ospilhacheia/tretadev#116]
Eu acho que os feios deveriam abrir a câmera, e os bonitos ficarem com a câmera fechada.

**Comment by izichtl** [comment]
Eu estava com a issue aberta para enviar essa, kkkk

Só psicopata abre amigo. 

Eu nunca abro porque sempre trabalho sem camisa (mas deixo uma camisa disponível caso tenha reunião com a gestão)

**Comment by efeeseao** [comment]
Eu não abro a câmera não tenho nada para mostrar às pessoas!

**Comment by JessicaNathany** [comment]
Minha opinião sobre. Eu acho que ja temos uma mordomia na área de TI em ganhar muito bem, trabalhar remoto e abrir uma câmera em uma reunião, é uma coisa que não tem preço. Não custa absolutamente nada. Não precisa abrir em todas, mas acho importante abrir. Porque você mostra que vc está presente naquele local, você consegue gesticular falando, sendo que 80% da nossa comunicação é linguagem corporal. Mundo corporativo é isso, precisamos saber se comunicar bem, falar com quem não gostamos negociar, convencer e não é com a câmera ligada que você vai conseguir isso. Desenvolvedores acham que o trampo é só codar e não é isso. Não somos robôs. Nós analisamos, criamos estratégias, codificamos, negociamos prazos, explicamos soluções, treinamos etc. E ter uma boa comunicação é crucial para se tornar um bom desenvolvedor(a). Tavez no futuro a pessoa queira desenvolver um produto ou criar algum curso ou vender sua ideia seu projeto e como vai fazer isso se não souber se comunicar bem?

Sou a favor de termos a câmera ligada. Imagina vc dev fazendo uma entrevista pra uma big tech com a câmera desligada, quais são as chances deles gostarem de vc?

**Comment by brunokim** [comment]
>  Imagina vc dev fazendo uma entrevista pra uma big tech com a câmera desligada, quais são as chances deles gostarem de vc?

Na real, vc nem faz entrevista sem câmera. A fraude em entrevistas tá tão absurda que exigem.

Eu falo muito com as mãos e não seria entendido sem estar com a câmera aberta. Eu acho **legal** ver a outra pessoa e pronto. Eu entendo quem não queira abrir na primeira reunião do dia às 08:00, mas ficar sempre vendo o avatar ou letra... cansa.

Também acho bem bosta um gestor bosta falar bosta de que tem ou não tem que fazer X pela preferência individual do bostinha, para qualquer X, se não afeta a realidade material do trabalho. Pior ainda se a pessoa nem é gestora, bosteando pelos outros.

Uma questão mais interessante: é de bom tom **fumar com a câmera aberta**? Você tá na sua casa, sem compartilhar fumaça com ninguém.

**Comment by izichtl** [comment]
Preciso assumir algo, diante das evidencias apresentadas no TT, essa semana eu fiz duas dailys com a camera aberta. 

Foi até bom sabia, tinha gente ali que nunca tinha visto minha cara.

**Comment by JessicaNathany** [comment]
> > Imagina vc dev fazendo uma entrevista pra uma big tech com a câmera desligada, quais são as chances deles gostarem de vc?
> 
> Na real, vc nem faz entrevista sem câmera. A fraude em entrevistas tá tão absurda que exigem.
> 
> Eu falo muito com as mãos e não seria entendido sem estar com a câmera aberta. Eu acho **legal** ver a outra pessoa e pronto. Eu entendo quem não queira abrir na primeira reunião do dia às 08:00, mas ficar sempre vendo o avatar ou letra... cansa.
> 
> Também acho bem bosta um gestor bosta falar bosta de que tem ou não tem que fazer X pela preferência individual do bostinha, para qualquer X, se não afeta a realidade material do trabalho. Pior ainda se a pessoa nem é gestora, bosteando pelos outros.
> 
> Uma questão mais interessante: é de bom tom **fumar com a câmera aberta**? Você tá na sua casa, sem compartilhar fumaça com ninguém.
eu tbm falo muito gesticulando e quando falo com alguém que está com a camera desligada, eu não sei o que a pessoa está sentindo naquele momento ou se ela está lá me ouvindo, pq não a estou vendo nem lendo suas expressões. Claro que, não falo que deixo ligada 100% do tempo, mas consideravelmente boa parte faço. Não tenho daily só KM que é duas vezes na semana então nem é ruim pra mim. Reuniões com discussões técnicas tbm deixo aberta.


Eu tbm falo mto desticulando e quando comecei a abrir a camera, eu era mais vista e mais lembrada. Sensação de presença saca, eu gosto. Tbm não curto falar com alguém ou uma discussão técnica e a pessoa ta com a câmera desliga pq não sei o a reação dela ou sua expressão, se é que ela está lá né. Mas tipo eu penso assim, eu não faço isso 100% do tempo só nas KMs ou reuniões importantes discussões técnicas e tal, dayle não tenho. Então não me incomoda ou me dói abrir ja trampo de casa e pra mim ta sussegado  :)
Sobre fumar de câmera aberta nunca fiz pq não fumo hasuhs mas não acho nada de mais.

**Comment by LukasLimalkl** [comment]
> Minha opinião sobre. Eu acho que ja temos uma mordomia na área de TI em ganhar muito bem, trabalhar remoto e abrir uma câmera em uma reunião, é uma coisa que não tem preço. Não custa absolutamente nada. Não precisa abrir em todas, mas acho importante abrir. Porque você mostra que vc está presente naquele local, você consegue gesticular falando, sendo que 80% da nossa comunicação é linguagem corporal. Mundo corporativo é isso, precisamos saber se comunicar bem, falar com quem não gostamos negociar, convencer e não é com a câmera ligada que você vai conseguir isso. Desenvolvedores acham que o trampo é só codar e não é isso. Não somos robôs. Nós analisamos, criamos estratégias, codificamos, negociamos prazos, explicamos soluções, treinamos etc. E ter uma boa comunicação é crucial para se tornar um bom desenvolvedor(a). Tavez no futuro a pessoa queira desenvolver um produto ou criar algum curso ou vender sua ideia seu projeto e como vai fazer isso se não souber se comunicar bem?
> 
> Sou a favor de termos a câmera ligada. Imagina vc dev fazendo uma entrevista pra uma big tech com a câmera desligada, quais são as chances deles gostarem de vc?

Concordo com vc, não custa nada abrir a camera pelo menos de vez em quando, agr toda hr com a webcam desligada n da
